### ISSUE TITLE
[Comprehension] User is uncertain where a button or link leads.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#77117]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Accredited Representation Management Team
**Product:** Find a Representative (MVP)
**Feature:** Find a Representative

## Findings details
**VA.gov Experience Standard - issue:** User is uncertain where a button or link leads.
**VA.gov Experience Standard - category:** Comprehension
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @erinrwhite (IA)

### Description

The address link for each result opens Google Maps in a new tab, without first warning users that will happen. The biggest impact is likely to be on screen reader users, who aren't always alerted by their computer that a new tab has opened. Keyboard shortcuts to go back won't work, which adds some extra friction to their experience.

## Recommended action

Three possible solutions here:
- Add text to the link indicating that it will open in a new tab, eg. `<a href="..." tabindex="0" target="_blank" rel="noreferrer">300 S State St  <br>Chicago, IL 60604 (opens in a new tab)</a> - **future concept**
- Add `sr-only` text indicating that it will open in a new tab, eg. `<a href="..." tabindex="0" target="_blank" rel="noreferrer">300 S State St  <br>Chicago, IL 60604 <span class="sr-only">(opens in a new tab)</span></a> - **will resolve in this ticket**
- Add an `aria-label` indicating that it will open in a new tab, eg. `<a href="..." tabindex="0" target="_blank" rel="noreferrer" aria-label="300 S State St Chicago, IL 60604 (opens in a new tab)">300 S State St  <br>Chicago, IL 60604 <span class="sr-only">(opens in a new tab)</span></a> - **will resolve in this ticket**

Visible text is usually the quickest way to make things accessible to everyone, but if that's not desirable for design reasons then providing it only for screen reader users is sufficient.

**Note from Product**: we will pursue the lower two recommendations for our 2.0 version.  We can discuss the design portion, and if that first recommendation is the best fit, and implement and updates at a later time (with a new, dedicated ticket).

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Link ](https://design.va.gov/components/form/label)
- **Design System Pattern:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by holdenhinkle** [comment]
Adding the text to the `aria-label` per this convo - https://dsva.slack.com/archives/C05L6HSJLHM/p1709835678820169
